src/test/java/io/jenkins/plugins/pipeline_cloudwatch_logs/IntegrationTest.java

```diff
@@ -36,19 +36,19 @@
 import org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl;
 import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
 import org.jenkinsci.plugins.workflow.job.WorkflowJob;
-import org.junit.ClassRule;
-import org.junit.Test;
-import static org.junit.Assume.assumeFalse;
 import org.junit.Before;
 import org.junit.Rule;
-import org.jvnet.hudson.test.BuildWatcher;
+import org.junit.Test;
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.JenkinsRule;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.not;
+import static org.junit.Assume.assumeFalse;
 
 public class IntegrationTest {
 
-    @ClassRule public static BuildWatcher buildWatcher = new BuildWatcher();
```

Comment:
Was making tests run much more slowly, since it is constantly rechecking the log, and CWL queries are rather slow.